lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -325,10 +325,14 @@ def services_data_for_country_and_partner_sex_and_residency?
     end
 
     def services_data_for_country_and_partner_sex?
-      @services_data.has_key?(ceremony_country) &&
+      services_data_for_country? &&
         @services_data[ceremony_country].has_key?(@sex_of_your_partner)
     end
 
+    def services_data_for_country?
+      @services_data.has_key?(ceremony_country)
+    end
+
```

Comment:
I think `services_data_for_country?` is currently more consistent with the other similar methods in the `MarriageAbroadCalculator` (see all the [`services_*?` methods in the calculator][calc-services-methods]).

I agree that it can be improved but am happy to leave it for another time.

[calc-services-methods]: https://github.com/alphagov/smart-answers/blob/master/lib/smart_answer/calculators/marriage_abroad_calculator.rb#L294